src/components/Beer.js

```diff
@@ -0,0 +1,57 @@
+import './beer.css'
+import { useParams  } from "react-router-dom"
+import { useEffect } from 'react'
+import { useState } from 'react/cjs/react.development'
+
+const Beer = ((props) => {
+
+    const { id } = useParams()
+
+    const [beer, setBeer] = useState()
+
+
+    let beerArray = props.beers.filter((beer)=>{
+        if (beer._id === id){
+            return true
+        } else {
+            return false
+        }
+    })
+
+    console.log(id)
+
+    useEffect( () => {
+        if (id === 'random'){
+            let random = Math.floor( Math.random() * props.beers.length)
+            setBeer(props.beers[random])
+            console.log(beer)
+        } else {
+            setBeer(beerArray.shift())
+            console.log(beer)
+        }
+    })
```

Comment:
Good solution, however according to instruction, you should have made call to axios and used endpoints to get singleBeer and randomBeer